packages/brain/src/modules/apiServers/launchpad/routes/keystores.ts

```diff
@@ -0,0 +1,65 @@
+import { Tag, Web3signerDeleteRequest } from "@stakingbrain/common";
+import express from "express";
+import { deleteValidators } from "../../../../calls/deleteValidators.js";
+import { importValidators } from "../../../../calls/importValidators.js";
+import { validateImportKeystoresRequestBody, validateDeleteRequestBody } from "../validation/requestValidation.js";
+import logger from "../../../logger/index.js";
+import { BrainKeystoreImportRequest } from "../types.js";
+
+const keystoresRouter = express.Router();
+
+const keystoresEndpoint = "/eth/v1/keystores";
```

Comment:
As you can see here where are proxying the existing api https://ethereum.github.io/keymanager-APIs/#/Local%20Key%20Manager/listKeys

I would also add some comments refering this standard

Reply:
Done